src/components/common/MetaTags/index.tsx

```diff
@@ -41,8 +41,6 @@ const MetaTags = ({ prefetchUrl }: { prefetchUrl: string }) => (
     {/* Favicons */}
     <link rel="shortcut icon" href="/favicons/favicon.ico" />
     <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png" />
-    <link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png" />
     <link rel="mask-icon" href="/favicons/safari-pinned-tab.svg" color="#000" />
```

Comment:
Do we not need to update these other favicons?

Reply:
I've added dark versions to the ones that were subject to poor contrast in browser's tabs.
If I get it correctly the ones I left in `<MetaTags>` are for other uses.

Reply:
I'm still seeing the old icon in Firefox:
![image](https://user-images.githubusercontent.com/20442784/207297325-ccd331e0-58e4-40f4-b464-00c91a231091.png)

I would suggest we test adding an icon to the home screen of an iPhone, as well as pinning a tab in Safari.

Reply:
I opened it in a Private Window though.